plane/source/docs/quadplane-frame-setup.rst

```diff
@@ -85,9 +88,21 @@ You can optionally move the quad motors to be on any other channel above
 
 Tricopter
 ---------
+Frame Type 7 is Tricopter and can be either non-Tiltrotor configuration, or :ref:`Tiltrotor<guide-tilt-rotor>` configured using either Vectored or Non-Vectored yaw control.
+
+With the default Tricopter frame the motors are on outputs
```

Comment:
this may be dependent on number of outputs on the controller....

Reply:
I would word this section (to added below existing):
The Tricopter frame will attempt to assign the following output functions, if not already assigned , to unassigned outputs 5 and above:
Motor1: Right Front, looking from above
Motor2: Left Front 
Motor4: Rear
Motor7: Rear motor tilt servo (not used in vectored yaw tilt-rotor configurations)

Reply:
> I would word this section (to added below existing):
> The Tricopter frame will attempt to assign the following output functions, if not already assigned , to unassigned outputs 5 and above:
I don't think that's correct. I tested disabling all outputs and it will definitely attempt to assign to output 1 if it is the first unassigned output. Another real example, the T1 Ranger doesn't have a rudder, so I disabled output 4 - as a result auto assign put Motor 4 on output 4 (i.e. < 5).

Reply:
all bets are off if you have altered output functions...plane sets outputs 1-4 by default, then qplane enable sets 5-8 since 1-4 are already set, then changing frame sets the appropriate outputs...ie hex adds 9 and 10, tri, drops 8 and adds 11...but if you have messed with things before you enable a frame, all bets are off and it will try to add motors as it can...